### PR TITLE
Archive sample

### PR DESCRIPTION
**Pull request by samantharamon**
Updates the sample structure to allow users to run the sample and archives the sample. Changes were not made to the code/logic itself.

Related PR: https://github.com/OfficeDev/office-js-docs-pr/pull/4335

### CODE REVIEW

File: manifest-localhost.xml [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+  </Hosts>
+  <Requirements>
+    <Sets>
+      <Set Name="Mailbox" MinVersion="1.1"/>
+    </Sets>
+  </Requirements>
+  <FormSettings>
+    <Form xsi:type="ItemRead">
+      <DesktopSettings>
+        <SourceLocation DefaultValue="https://localhost:3000/module.html"/>
```
Comment (davidchesnut) [inline manifest-localhost.xml]:
```suggestion
        <SourceLocation DefaultValue="https://localhost:3000/src/module/module.html"/>
```

File: manifest-localhost.xml [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+          <bt:Image id="AssociateIcon.80x80" DefaultValue="https://localhost:3000/assets/associate-80.png"/>
+          <bt:Image id="PartnerIcon.16x16" DefaultValue="https://localhost:3000/assets/partner-16.png"/>
+          <bt:Image id="PartnerIcon.32x32" DefaultValue="https://localhost:3000/assets/partner-32.png"/>
+          <bt:Image id="PartnerIcon.80x80" DefaultValue="https://localhost:3000/assets/partner-80.png"/>
+          <bt:Image id="ExecutiveIcon.16x16" DefaultValue="https://localhost:3000/assets/executive-16.png"/>
+          <bt:Image id="ExecutiveIcon.32x32" DefaultValue="https://localhost:3000/assets/executive-32.png"/>
+          <bt:Image id="ExecutiveIcon.80x80" DefaultValue="https://localhost:3000/assets/executive-80.png"/>
+        </bt:Images>
+        <bt:Urls>
+          <bt:Url id="Module.Url" DefaultValue="https://localhost:3000/module.html"/>
```
Comment (davidchesnut) [inline manifest-localhost.xml]:
```suggestion
          <bt:Url id="Module.Url" DefaultValue="https://localhost:3000/src/module/module.html"/>
```

**Review by davidchesnut — CHANGES_REQUESTED** [review]
Just a couple changes to make the local manifest work correctly.

File: manifest-localhost.xml (hunk 1 above)
Comment (samantharamon) [reply manifest-localhost.xml]: Thanks for catching this!

**Review by davidchesnut — APPROVED** [review]
Looks good!

**Pull request merged**